doc/conf.py

```diff
@@ -82,15 +84,27 @@
     "announcement": announcement_text,
 }
 
-extensions += [
-    'sphinx.ext.napoleon',
-    'nbsite.gallery',
+
+extensions = [
```

Comment:
Wouldn't it be easy to remove the one you don't like?

Reply:
If I was just removing one of them yes, but with two I find this nicer/more explicit.